src/controllers/controllermanager.cpp

```diff
@@ -174,7 +178,20 @@ void ControllerManager::updateControllerList() {
 
     locker.relock();
     if (newDeviceList != m_controllers) {
+        // Check for keyboard controller
+        KeyboardController* pKeyboardController;
+        foreach (Controller* pController, newDeviceList) {
```

Comment:
If you have fun .... ;-)
If not, no it is not an issue, but you should use the new loop for new code.